repose-aggregator/src/docs/asciidoc/filters/cors.adoc

```diff
@@ -26,18 +33,45 @@ All of these headers will be supplied by the client making the request.
 | None required
 a| * `Origin`
    * `Access-Control-Request-Method`
-a| * `Origin`
+a| * `Origin`*
 |===
 
-=== Created
-The CORS filter wil not add any headers to the response for non-CORS requests.
-The filter will add the following headers to the response for these scenarios when the request is allowed to proceed:
+*{sp}If the `Origin` header matches the original `Host` header in the request, it will be considered a non-CORS request.
```

Comment:
So it renders the footnote within the table cell. How annoying. It seems I'm not alone in thinking that.
https://github.com/asciidoctor/asciidoctor/issues/1705

However, that could probably be customized in our CSS?